docs/nl/docs/features.md

```diff
@@ -0,0 +1,201 @@
+class User(BaseModel):
+    id: int
+    name: str
+    joined: date
+```
+
+Vervolgens kan je het op deze manier gebruiken:
+
+```Python
+my_user: User = User(id=3, name="John Doe", joined="2018-07-19")
+
+second_user_data = {
+    "id": 4,
+    "name": "Mary",
+    "joined": "2018-11-30",
+}
+
+my_second_user: User = User(**second_user_data)
+```
+
+/// info
+
+`**second_user_data` betekend:
+
+Geef de keys en values van de `second_user_data` dict direct door als key-value argumenten, gelijk aan: `User(id=4, name=“Mary”, joined=“2018-11-30”)`
+
+///
+
+### Editor-ondersteuning
+
+Het gehele framework is ontworpen om eenvoudig en intuïtief te zijn in gebruik. Alle beslissingen zijn getest op meerdere code-editors nog voordat het daadwerkelijke ontwikkelen begon, om zo de beste ontwikkelervaring te garanderen.
+
+Uit enquêtes onder Python ontwikkelaars blijkt maar al te duidelijk dat "autocompletion" <a href="https://www.jetbrains.com/research/python-developers-survey-2017/#tools-and-features" class="external-link" target="_blank">een van de meest gebruikte functionaliteiten is</a>.
+
+Het hele **FastAPI** framework is daarop gebaseerd. Autocompletion werkt overal.
+
+Je hoeft zelden terug te vallen op de documentatie.
+
+Zo kan je editor je helpen:
+
+* in <a href="https://code.visualstudio.com/" class="external-link" target="_blank">Visual Studio Code</a>:
+
+![editor ondersteuning](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+* in <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a>:
+
+![editor ondersteuning](https://fastapi.tiangolo.com/img/pycharm-completion.png)
+
+Je krijgt autocomletion die je voorheen misschien zelfs voor onmogelijk had gehouden. Zoals bijvoorbeeld de `price` key in een JSON body (die genest had kunnen zijn) die afkomstig is van een request.
+
+Je hoeft niet langer de verkeerde keys in te typen, op en neer te gaan tussen de documentatie, of heen en weer te scrollen om te checken of je `username` of toch `user_name` had gebruikt.
+
+### Kort
+
+Dit framework heeft voor alles verstandige **standaardinstellingen**, met overal optionele configuraties. Alle parameters kunnen worden verfijnd zodat het past bij wat je nodig hebt, om zo de API te kunnen definiëren die jij nodig hebt.
+
+Maar standaard werkt alles **“gewoon”**.
+
+### Validatie
+
+* Validatie voor de meeste (of misschien wel alle?) Python **datatypes**, inclusief:
+    * JSON objecten (`dict`).
+    * JSON array (`list`) die itemtypes definiëren.
+    * String (`str`) velden, die min en max lengtes hebben.
+    * Getallen (`int`, `float`) met min en max waarden, etc.
+
+* Validatie voor meer exotische typen, zoals:
+    * URL.
+    * E-mail.
+    * UUID.
+    * ...en anderen.
+
+Alle validatie wordt uitgevoerd door het beproefde en robuuste **Pydantic**.
+
+### Beveiliging en authenticatie
+
+Beveiliging en authenticatie is geïntegreerd. Zonder compromissen te doen naar databases of datasmodellen.
```

Comment:
```suggestion
Beveiliging en authenticatie is geïntegreerd. Zonder compromissen te doen naar databanken of datamodellen.
```

Reply:
Isn't database also a Dutch word? I think it's in the dictionary.

Reply:
Maybe, Dutch has a lot of English "loan words", but I think "databank" is a very clear Dutch word that is used a lot in Dutch text books and other literature, so I would definitely prefer it over "database".

Reply:
I think you are right. The Dutch spoken in The Netherlands uses more English words in comparison to the Dutch spoken in for example Belgium.

 At universities in The Netherlands, Dutch spoken computer science (informatica) programs use "databases".

- https://www.uu.nl/bachelors/informatica/studieprogramma
- https://www.uva.nl/programmas/bachelors/informatica/studieprogramma/studieprogramma.html

> e.g. first site I look, informatica opleiding at UGent, has a course called "databanken": https://informatica.ugent.be/bachelor/

Yes that's indeed the case. However, "Databases" is also the name of several courses at Belgian universities.

- https://www.uantwerpen.be/nl/studeren/aanbod/alle-opleidingen/informatica-studeren/bachelor/studieprogramma/
- https://www.uhasselt.be/nl/studeren/opleidingen/bachelor-en-master-in-de-informatica#anch-215-bachelor

The most recent edition of the Dutch book ""Principes van databanken" by Guy de Tré that is used in the UGent course is published with the title "Principes van databases".

The search term for "database" is also more popular than "databank" in both Belgium and The Netherlands.

- https://trends.google.com/trends/explore?date=all&geo=BE&q=database,databank&hl=nl-NL
- https://trends.google.com/trends/explore?date=all&geo=NL&q=database,databank&hl=nl-NL

So I don't really think there is a correct definitive correct Dutch translation of "databases".

However, in my view, the term “database” is more often used in Dutch (in Belgium and The Netherlands, in Universities, in text books and in government).

I think "database" leads to less confusion for the user when also taking into account that they are probably also reading other (English) documentation.

Reply:
> So I don't really think there is a correct definitive correct Dutch translation of "databases".

Haha - I respectfully disagree: the clear correct Dutch translation of "database" is "databank", but the question is whether the actual Dutch translation, or the English loan word, is used more frequently in Dutch texts 🤷 

Anyway, why don't we adhere to this rule of thumb in cases like these: to follow the Dutch-speaking Wikipedia? While that's usually also a bit skewed towards The Netherlands vs Belgium, at least it's an "authorative" source to use as gold truth.

In this particular case: that would mean we stick to "database" as https://nl.wikipedia.org/wiki/Databank redirects to https://nl.wikipedia.org/wiki/Database.

Please do go through my reviewer's comments carefully though - some of the "databank" suggestions also contain fixes for other words 😉